deployment/helm/resource-policy/topology_aware_values.yaml

```
@@ -0,0 +1,42 @@
+# Default values for nri-plugins.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+---
+image:
+  name: ghcr.io/containers/nri-plugins/nri-resource-policy-topology-aware
+  # tag, if defined will use the given image tag, otherwise Chart.AppVersion will be used
+  tag: unstable
+  pullPolicy: Always
+container:
+  name: nri-resource-policy-topology-aware
+
+namespace: kube-system
+
+configMap:
+  data:
+    ReservedResources:
+      cpu: 750m
```

Can we have simply

```suggestion
config:
  ReservedResources:
    cpu: 750m
```

No need to replicate the podspec/template structure here. WDYT?

---

good point. Fixed